python/ql/src/experimental/dataflow/internal/TaintTrackingPublic.qll

```diff
@@ -6,24 +6,20 @@
 private import python
 private import TaintTrackingPrivate
 private import experimental.dataflow.DataFlow
-
 // /**
 //  * Holds if taint propagates from `source` to `sink` in zero or more local
 //  * (intra-procedural) steps.
 //  */
 // predicate localTaint(DataFlow::Node source, DataFlow::Node sink) { localTaintStep*(source, sink) }
-
 // // /**
 // //  * Holds if taint can flow from `e1` to `e2` in zero or more
 // //  * local (intra-procedural) steps.
 // //  */
 // // predicate localExprTaint(Expr e1, Expr e2) {
 // //   localTaint(DataFlow::exprNode(e1), DataFlow::exprNode(e2))
 // // }
-
 // // /** A member (property or field) that is tainted if its containing object is tainted. */
 // // abstract class TaintedMember extends AssignableMember { }
-
```

Comment:
:(

Reply:
Oh, huh. I guess those empty lines really ought to have had empty comments as well.